### ISSUE TITLE
mysql-common installs /etc/mysql/my.cnf - mariadb-common insists its a symlink

### ISSUE THREAD

**Issue by grooverdan** [ottok/mariadb-10.0#10]
dpkg -L mysql-common
/.
/usr
/usr/share
/usr/share/doc
/usr/share/doc/mysql-common
/usr/share/doc/mysql-common/changelog.Debian.gz
/usr/share/doc/mysql-common/copyright
/etc
/etc/mysql
/etc/mysql/my.cnf
/etc/mysql/conf.d
/etc/mysql/conf.d/.keepme

 dpkg -l | fgrep mysql-common
ii  mysql-common                   5.6.21-25.6-1             all          MySQL database common files, e.g. /etc/mysql/my.cnf

when installing mariadb-common....

Setting up mariadb-common (10.0.17-1~exp1) ...
Notice: configure-symlinks trigger could not be called.
Please manually make /etc/mysql/my.cnf a symlink that points to mariadb.cnf.

**Comment by ottok** [comment]
This is due to a symlink decoupling in Debian, but the mysql-common that supports it has only been released in Ubuntu, not Debian yet. Where is you mysql-common from? Does it work if you simply run
```
sudo rm -rf /etc/mysql/my.cnf
sudo ln -s /etc/mysql/mariadb.cnf /etc/mysql/my.cnf
```

**Comment by grooverdan** [comment]
yes I did those commands to get it to work. As long as work is in progress to remove manual commands in the install i'm happy.

**Comment by ottok** [comment]
I got tried of waiting for somebody to upload the new mysql package into Debian experimental that would provide the correct configure-symlink facility so I made a fully automatic temporary (and maybe even permanent) fix in commit 90b6bde6
